docs/docsite/rst/porting_guides/porting_guide_2.9.rst

```diff
@@ -66,6 +66,7 @@ No notable changes
 Noteworthy module changes
 -------------------------
 
+* :ref:`vmware_cluster <vmware_cluster_module>` has been refactored into several specialized modules; the functionality to configure DRS, HA and vSAN has been moved to :ref:`vmware_cluster_drs <vmware_cluster_drs_module>`, :ref:`vmware_cluster_ha <vmware_cluster_ha_module>` and :ref:`vmware_cluster_vsan <vmware_cluster_vsan_module>`.
```

Comment:
I'd add a short statement about the benefit to users (or to developers) of this change, and make it clear that the old module still exists. I don't know what the motivation for this change was, but I'll give an example - if the idea was to make maintenance easier, you could say:
```
* :ref:`vmware_cluster <vmware_cluster_module>` was refactored for easier maintenance/bugfixes. The original module only adds/removes clusters. Use the three new, specialized modules to configure clusters. Configure DRS with :ref:`vmware_cluster_drs <vmware_cluster_drs_module>`, HA with :ref:`vmware_cluster_ha <vmware_cluster_ha_module>`, and vSAN with :ref:`vmware_cluster_vsan <vmware_cluster_vsan_module>`.
```

Reply:
Yes, the idea was to make maintenance easier. I've copied your suggestion.